src/view/hocs/withYfmHtml/useYfmHtmlBlockRuntime.ts

```diff
@@ -0,0 +1,4 @@
+/** @internal */
+export function useYfmHtmlBlockRuntime() {
+    import(/* webpackChunkName: "yfm-html-runtime" */ '@diplodoc/html-extension/runtime');
+}
```

Comment:
Let's add some flag that determines whether to download runtime: always or by meta

Reply:
fixed